tests/utils/test_cache_utils.py

```diff
@@ -643,10 +643,10 @@ def test_dynamic_cache_exportability(self):
             past_key_values=past_key_values_eager,
             use_cache=True,
         )
-        self.assertTrue(torch.allclose(res.logits, res_eager.logits, atol=1e-5))
+        self.assertTrue(torch.allclose(res.logits, res_eager.logits, atol=1e-7))
```

Comment:
running on CI runners, and it passes.

I would like to hear from @sywangyi why the atol is increased in #39412, maybe this is some other test cases need this larger atol.

Reply:
But did you observe larger atol is necessary when you worked on that PR? I am testing in our CI runners and it works with 1e-7. I am just wondering what is the motivation of increasing to 1e-5 (I saw you even change it to 1e-3 once from the commit history).

@st81 I personally would not be bother of keeping it as 1e-5 however.

Reply:
Understood. Just a relevant thread if you are interested to read

https://github.com/pytorch/pytorch/issues/157274#issuecomment-3020268707

that is one reason I might keep it as 1e-5 now